webapp/src/containers/modals/SubscribeModal/index.tsx

```diff
@@ -108,6 +112,14 @@ const SubscribeModal = () => {
         }
     };
 
+    useEffect(() => {
+        if (formFields.serviceType === 'board') {
```

Comment:
constants for board and repos

Reply:
Fixed [here](https://github.com/Brightscout/mattermost-plugin-azure-devops/pull/81)